CHANGELOG

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## UNRELEASED
+[0.8.1] - 2023-04-14
```

Comment:
Should be just under UNRELEASED rather than a version bump.

Reply:
Removed version bump